docs/build/micro-rdk/data_management/_index.md

```diff
@@ -0,0 +1,176 @@
+
+### Supported components
+
+Only the following components types are currently supported with data capture:
+|Type | Method |
+|----- | ----------- |
+|Sensor | GetReadings |
+|Movement Sensor | AngularVelocity, LinearAcceleration, LinearVelocity |
+
+To add data capture for a component, navigate to the **CONFIGURE** tab of your machine's page in the Viam app.
+
+For each component you can capture data for, there is a `Data capture` section in its panel.
+Click `Add Method` and then select the **Method** type and the capture **Frequency**.
+
+{{< alert title="Caution" color="caution" >}}
+
+Avoid configuring data capture to higher rates than your hardware can handle, as this leads to performance degradation.
+
+{{< /alert >}}
+
+Click the **Save** button in the top right corner of the page.
+
+Now your data will be captured at the configure frequency and synced to Viam cloud at the selected interval.
+
+{{%expand "Click to view an example JSON configuration capturing data from the GetReadings method of a temperature sensor and wifi signal sensor" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "services": [
+    {
+      "attributes": {
+        "capture_dir": "",
+        "tags": [],
+        "additional_sync_paths": [],
+        "sync_interval_mins": 3
+      },
+      "name": "dm",
+      "namespace": "rdk",
+      "type": "data_manager"
+    }
+  ],
+  "components": [
+    {
+      "type": "sensor",
+      "model": "tmp36",
+      "attributes": {
+        "analog_reader": "temp",
+        "num_readings": 15
+      },
+      "depends_on": [],
+      "service_configs": [
+        {
+          "attributes": {
+            "capture_methods": [
+              {
+                "capture_frequency_hz": 0.2,
+                "additional_params": {},
+                "method": "Readings"
+              }
+            ]
+          },
+          "type": "data_manager"
+        }
+      ],
+      "name": "tmp36",
+      "namespace": "rdk"
+    },
+    {
+      "type": "sensor",
+      "model": "wifi-rssi",
+      "attributes": {},
+      "service_configs": [
+        {
+          "type": "data_manager",
+          "attributes": {
+            "capture_methods": [
+              {
+                "additional_params": {},
+                "method": "Readings",
+                "capture_frequency_hz": 0.1
+              }
+            ]
+          }
+        }
+      ],
+      "name": "my-wifi-sensor",
+      "namespace": "rdk"
+    }
+  ]
+}
+```
+
+{{% /expand%}}
+
+- To enable or disable data capture for a configured component or method, use the `on/off` toggle.
+- To change the frequency of data capture for a method, enter the number of measurements you wish to capture per second in the frequency field.
+
+After adding configuration for the methods, click the **Save** button in the top right corner of the page.
+
+If you want to remove a capture method from the configuration, click the `delete` icon.
```

Comment:
Should this be a bullet point after change freq and enable/disable? It seems like the logical third bullet point, with "after adding, save" then being the final line in this section.